xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties

```diff
@@ -2109,6 +2109,9 @@ platform.core.rightsManagement.editRightsForSpace=Editing access rights for spac
 platform.core.rightsManagement.ajaxFailure=An error occurred while communicating with the server. Please check that the server is accessible, and you have the proper rights to perform the requested action.
 platform.core.rightsManagement.saveFailure=An exception occurred while trying to save the current modifications. Please check if you have the proper rights to perform these modifications.
 platform.core.rightsManagement.saveComment={0} {1} right for {2}
+platform.core.rightsManagement.allowed=Allowed is the current state. Switch to denied.
+platform.core.rightsManagement.denied=Denied is the current state. Switch to undefined.
+platform.core.rightsManagement.undefined=Undefined is the current state. Switch to allowed.
```

Comment:
What do you think about something like "Allowed. Click to deny."/"Denied. Click to switch to undefined."/"Undefined. Click to allow." I'm not sure about the exact wording, either, but I think it could be interesting to have a shorter text when possible.